command/daemon/config.go

```diff
@@ -68,8 +68,13 @@ type Config struct {
 	Resources struct {
 		LimitCPU      int64     `envconfig:"DRONE_RESOURCE_LIMIT_CPU"`
 		LimitMemory   BytesSize `envconfig:"DRONE_RESOURCE_LIMIT_MEMORY"`
-		RequestCPU    int64     `envconfig:"DRONE_RESOURCE_REQUEST_CPU"`
-		RequestMemory BytesSize `envconfig:"DRONE_RESOURCE_REQUEST_MEMORY"`
+		RequestCPU    int64     `envconfig:"DRONE_RESOURCE_REQUEST_CPU" default:"100"`
```

Comment:
@bradrydzewski I have set default stage request as 100m cpu and 100Mib memory. Since stage resources are newly introduced, old pipeline will face issues if DRONE_RESOURCE_REQUEST_CPU or DRONE_RESOURCE_REQUEST_MEMORY is not set in the kube-runner. Let me know if default values needs to be changed.